public/openapi.yaml

```diff
@@ -152,77 +153,23 @@ components:
             description: more information about the iteration data file
             url: >-
               https://learning.postman.com/docs/running-collections/working-with-data-files/
+    CollectionRunJsonOutput:
```

Comment:
You can put these lines 157 to  165 directly in the form-data ( i.e. replace line 116 by them)
Otherwise , rename this schema the "OutputSummaryFile" (line 156)